spec/spec_helper.rb

```diff
@@ -5,6 +5,7 @@
 require "docker"
 require "rest-client"
 require "active_support/core_ext/module/delegation"
+require "pry"
```

Comment:
Accident?

Reply:
Nope! I think this is a useful thing to have loaded in specs.

Reply:
👍